aiohttp/client_reqrep.py

```diff
@@ -1238,6 +1238,17 @@ async def read(self) -> bytes:
             await self._wait_released()  # Underlying connection released
         return self._body
 
+    async def discard_content(self) -> None:
```

Comment:
I'm thinking the potential for using this in the wrong circumstance may mean it's better not to put it in as a public API? Maybe just manually do this in a couple of places that should be optimised for this case?